Formula/ooniprobe.rb

```diff
@@ -169,10 +167,6 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources", "--update-inputs", "--update-geoip"
```

Comment:
Oh I see. I didn't think about the bottles.

Will add it back then and take note of the fact that in the future ooniprobe should support disabling the fetching of resources via setup.py for use-cases like this.